callisto_core/requirements/base.txt

```diff
@@ -1,7 +1,9 @@
 argon2_cffi==16.3.0
+celery>=4.1.0
 django==2.0
 django-polymorphic==1.3
 django-ratelimit==1.1.0
+django_celery_results>=1.0.1
```

Comment:
close, but opposite!

they should be `==` in `requirements/*`

and `>=` in `setup.py`